### ISSUE TITLE
Allow submissions after assignment lock date if extension present

### ISSUE THREAD

**Issue by colinschoen** [Cal-CS-61A-Staff/ok#1181]
Currently a student is unable to submit an assignment past the assignment lock date even if they have an extension. An extension should take priority over the lock date and a submission should be allowed in this case.

**Comment by Marlysson** [comment]
There are more details about this feature...

**Comment by colinschoen** [comment]
^ Can you rephrase @Marlysson ...

**Comment by ethanlee16** [comment]
I wasn't able to reproduce this (using the web client for submitting) after setting a past lock date and assigning an extension. Does this bug only apply to the Ok [Python client](https://github.com/Cal-CS-61A-Staff/ok-client) or has it been resolved?
